akvo/rsr/usecases/change_project_parent.py

```diff
@@ -134,12 +114,10 @@ def change_parent(project, new_parent, reimport=False, verbosity=0):
             model.objects.filter(id__in=[item_id]).update(**{f"{parent_attr}_id": target_id})
     if verbosity > 0:
         print(f"Change project {project.title} (ID:{project.id}) parent to {new_parent.title} (ID:{new_parent.id})")
-    RelatedProject.objects.filter(
-        project=old_parent, related_project=project, relation='2'
-    ).update(project=new_parent)
-    RelatedProject.objects.filter(
-        related_project=old_parent, project=project, relation='1'
-    ).update(related_project=new_parent)
+
+    # Set the new parent and update the descendants
+    project.set_parent(new_parent, True, update_descendants=True)
```

Comment:
`.save()`?

Reply:
Fixed in 086c9ca39ed0327d2c6b33a91903fd69b88318bf